homeassistant/components/rachio/binary_sensor.py

```diff
@@ -70,6 +78,18 @@ def _poll_update(self, data=None) -> bool:
         """Request the state from the API."""
         pass
 
+    @property
+    def device_info(self):
+        """Return the device_info of the device."""
+        return {
+            "identifiers": {(DOMAIN_RACHIO, self._controller.serial_number,)},
+            "connections": {
+                (device_registry.CONNECTION_NETWORK_MAC, self._controller.mac_address,)
```

Comment:
This is cool, because now other integrations can add data to this device, like Unifi can tell you if it's online/offline.